### ISSUE TITLE
Detect file changes in watch-folder over network fails

### ISSUE THREAD

**Issue by DasPflanze** [coppit/docker-handbrake#1]
Hello,

on my docker machine I got a media-share folder mounted from my NAS with NFS. Now I wan't to push files to this folder with my main-pc but monitor.py won't detect the changes. So I need to change the file on the docker-machine with "mv xyz_12_03_2017.ts xyz.ts" to get HandBrake running automatically.  Is it not possible to do or is there a bug with the nfs share?

Greets!

**Comment by coppit** [comment]
Yeah, I'm using inotify which only works for local paths. I may eventually switch to something more primitive that can handle network drive and other mounted drives.

**Comment by Symbiot78** [comment]
I think I have the same issue. I am running the container in bridged mode and it doesn't seem to detect any new content in the watch folder.. I can manually browse to the files and start it but nothing happens automatically.

Is this something you will change?

**Comment by coppit** [comment]
Bridge mode isn't relevant. What's relevant is whether the folder being watched is on the local filesystem of the host, or located on a networked drive that is mounted in the host. I'd have to rewrite the change detection engine to support network mounts.
